rijsdijk2021reproducing/openreview/packages.tex

```diff
@@ -0,0 +1,26 @@
+\usepackage[final]
+
+\usepackage[utf8]{inputenc} % allow utf-8 input
+\usepackage{hyperref}       % hyperlinks
+\usepackage{url}            % simple URL typesetting
+\usepackage{booktabs}       % professional-quality tables
+\usepackage{amsfonts}       % blackboard math symbols
+\usepackage{nicefrac}       % compact symbols for 1/2, etc.
+\usepackage{microtype}      % microtypography
+\usepackage{graphicx}
+
+\usepackage{subcaption}
+\captionsetup{compatibility=false}
+
+\usepackage[normalem]{ulem}
+\newif{\ifhidecomments}
+\usepackage{notes}
+
+\usepackage[
```

Comment:
Any reason why you need to add a bibliography package like this?

Reply:
apparently not.